spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-local/src/main/java/org/springframework/cloud/data/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -77,9 +79,11 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		catch (Exception e) {
 			throw new IllegalStateException("failed to determine URL for module: " + module, e);
 		}
-		args.add(String.format("--%s.server.port=%d", module, port));
-		args.add(String.format("--%s.endpoints.shutdown.enabled=true", module));
-		launcher.launch(new String[] { module }, args.toArray(new String[args.size()]));
+		args.put("server.port", String.valueOf(port));
```

Comment:
When dealing with a ModuleLaunchRequest directly, which is what this deployer does, there is no qualification.